src/docs/arch-rts.dox

```diff
@@ -65,11 +65,11 @@ store is empty and the stable timestamp is 10:
 
 Update list: U3 (30) -> U2 (20) -> U1 (10)\n
 On-disk: U2 (20)\n
-History store: Empty\n
+History store: U1 (10)\n
 Stable timestamp: 10\n\n
 
-RTS will mark all the unstable updates from the update list as aborted and the on-disk value will be
-replaced the most recent stable update after reconciliation:
+RTS will mark all the unstable updates from the update list as aborted. The on-disk value will be
+replaced by the most recent stable update after reconciliation, in this case from the history store:
```

Comment:
`in this case from the history store` the stable update already exists in the update list. It doesn't retrieve it from the history store. Removing this statement would be good.

Reply:
Done, thanks